test/passing/tests/cases_exp_grouping.ml.ref

```diff
@@ -2,92 +2,101 @@ let _ =
   match x with
   | A -> begin match B with A -> fooooooooooooo end
   | A -> begin match B with A -> fooooooooooooo | B -> fooooooooooooo end
-  | A -> begin
-    match B with
-    | A -> fooooooooooooo
-    | B -> fooooooooooooo
-    | C -> fooooooooooooo
-    | D -> fooooooooooooo
-  end
```

Comment:
This shows that for many values of `break-cases`, the `end` is aligned with the `|` of the pattern, which is ambiguous.